### ISSUE TITLE
Spanish version

### ISSUE THREAD

**Issue by lapin7** [rchain/Members#263]
**Goal:** a [. srt file](https://docs.google.com/document/d/16It5_3fghkhsvGWNVX9MswxPnMlvXCQ0w6aiKotrCxU/edit) 
stored in  [/RChain/Members/Marketing-Community Dev/YouTube](https://drive.google.com/drive/folders/1zBZXvPMIC9yBneYI2c4x-xGwr-D1X1Ii) 
with a translated transcript in Spanish of the file 
[[TRANSCRIPT][.srt][English] "Rchain Exclusive chat Lawrence Lerner & Nash Foster"](https://docs.google.com/document/d/1oTZVkYEhhnZtyU4DmNiiT3N39R0IVpHI92vP8kaltFs/edit)
**Progress indicator**: see the evolvement of the doc
**TIme**: 3 weeks
**Budget**: $5000
**Collaborator:** @kaka56

**Comment by Keaycee** [comment]
@kaka56 I can assist in doing this.

**Comment by kaka56** [comment]
@ keaycee i have gone far and am almost  through.

**Comment by kaka56** [comment]
@Keaycee you are invited to collaborate with me

**Comment by kaka56** [comment]
@Keaycee  here is the collaborative file https://docs.google.com/document/d/1ONyxaJTOL_ZXaFYQWVJ5dEUrnbTm-rCNW10f39sAIXY/edit
there is more to it .to review, edit/correct and  arrange in order.

**Comment by Keaycee** [comment]
@kaka56 I'm still proofreading the document.

**Comment by Viraculous** [comment]
@kaka56  can I also collaborate in editing for better precision?

**Comment by kaka56** [comment]
@Viraculous you are invited  to review

**Comment by ICA3DaR5** [comment]
So this is finished?

**Comment by kaka56** [comment]
@ICA3DaR5  Yes we are through  with  the Spanish  version and  changed  the  draft to Final.
